queries.sql

```diff
@@ -72,3 +72,75 @@ between '1990-1-1' and '2000-1-1'; --0
 --escape_attempts
 --------------------
  --2.6666666666666667
+
+update animals set species_id = 2 where name like '%mon';
+update animals set species_id = 1 where name not like '%mon';
```

Comment:
- [ ] In this query it ends with `WHERE species_id IS NULL. ` It doesn't include %mon.

Reply:
> * [ ]  In this query it ends with `WHERE species_id IS NULL. ` It doesn't include %mon.

@sja-thedude Which of the queries are you referring to? 76 or 77?
if it is 77, I believe both will result in the same outcome.